staking/types/messages.go

```diff
@@ -49,10 +50,10 @@ func (d Directive) String() string {
 type NewValidator struct {
 	Description       `json:"description" yaml:"description"`
 	CommissionRates   `json:"commission" yaml:"commission"`
-	MinSelfDelegation *big.Int           `json:"min_self_delegation" yaml:"min_self_delegation"`
-	StakingAddress    common.Address     `json:"staking_address" yaml:"staking_address"`
-	PubKey            shard.BlsPublicKey `json:"validating_pub_key" yaml:"validating_pub_key"`
-	Amount            *big.Int           `json:"amount" yaml:"amount"`
+	MinSelfDelegation *big.Int       `json:"min_self_delegation" yaml:"min_self_delegation"`
+	StakingAddress    common.Address `json:"staking_address" yaml:"staking_address"`
+	PubKey            bls.PublicKey  `json:"validating_pub_key" yaml:"validating_pub_key"`
```

Comment:
don't change this. we always use 48 bytes for pubkey, don't use the struct object.